### DISCUSSION TITLE
How to create child theme

### DISCUSSION THREAD

**Discussion by rrdgcoks** [OrchardCMS/OrchardCore#11448] [Q&A] [answered]
The only source for direction I have in creating a child theme is from here: https://docs.orchardcore.net/projects/O1/en/rtd/Documentation/Customizing-the-default-theme/

However, the docs seem to be outdated. The instructions involve using Code Generation. I've looked for anything similar in Admin but cannot find it. I tried the commands in Visual Studio Package Manage Console and PowerShell, hoping they might work.

The docs talk about "Theme Machine" but I can't find this either. 

Any pointers?

**Comment by Skrypt** [comment, marked as answer]
You are using the documentation for Orchard 01.

https://docs.orchardcore.net/en/latest/docs/getting-started/theme/
The Orchard Core documentation is here : https://docs.orchardcore.net/en/latest/docs/guides/create-admin-theme/

The theme inheritance is explained in the create admin theme doc.

**Reply by rrdgcoks** [reply]
Thank you!
